addons/account/static/src/components/actionable_errors/actionable_errors.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { Component, xml } from "@odoo/owl";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { useRecordObserver } from "@web/model/relational_model/utils";
+
+export class ActionableErrors extends Component {
+    static props = { ...standardFieldProps };
+    static template = xml`
+    <t t-name="account.ActionableErrors">
+        <t t-if="errorsData">
+            <div class="o_actionable_errors" role="alert">
+                <t t-foreach="errorsData" t-as="error" t-key="error">
+                    <div>
+                        <t t-out="error_value.message"/>
+                        <t t-if="error_value.action_name">
+                            <a href="#"
+                               t-on-click.prevent="() => this.handleOnClick(error_value)"
+                               t-att-actionName="error_value.action_name"
+                               t-att-actionParams="error_value.action_params">
+                                <i class="oi oi-arrow-right"/>
+                                <span><t t-out="error_value.action_text"/></span>
+                                <t t-if="error_value.critical"><i class="fa fa-warning"/></t>
+                            </a>
+                        </t>
+                    </div>
+                </t>
+            </div>
+        </t>
+    </t>`;
+
+    setup() {
+        useRecordObserver(this.update.bind(this));
```

Comment:
@h4818 This is enough?

Reply:
> @h4818 This is enough?

It should be enough, if the record changes, the `update` method will be called. It doesn't seem to be a reactive component, so there shouldn't be any need for `useState`

Reply:
> if this is only for the send & print wizard, does it even need the recordObserver?
It's not, it's also for Kenya, Intrastat and Saf-T.

@jco-odoo wants to use the state because he wants to use the widget in a computed field, showing warnings on the move's header.

Reply:
If Josse wants something, he will adapt this widget to his likings in his own PR.